packages/api-explorer/src/components/DocSource/DocSource.spec.tsx

```diff
@@ -89,12 +89,10 @@ describe('DocSource', () => {
       declaration.line
     )
     expect(link.closest('a')).toHaveAttribute('href', expected)
-    await waitFor(() => {
-      userEvent.hover(link)
```

Comment:
Just for clarity on what the previous code would be doing, is that on every failure and retry of the waitFor() the link would have been hovered. Doing this outside of the await would be the more correct way to do this (I think?). Moot point anyways.

Reply:
We should have a follow-up ticket to remove all `waitFor` statements we can. These changes were just those necessary to unblock this PR